github_archive/README.md

```diff
@@ -0,0 +1,495 @@
+            "open_issues_count": 3546,
+            "license": {
+              "key": "other",
+              "name": "Other",
+              "spdx_id": "NOASSERTION",
+              "url": null,
+              "node_id": "MDc6TGljZW5zZTA="
+            },
+            "allow_forking": true,
+            "visibility": "public",
+            "forks": 20670,
+            "open_issues": 3546,
+            "watchers": 56690,
+            "default_branch": "master"
+          }
+        },
+        "_links": {
+          "self": {
+            "href": "https://api.github.com/repos/elastic/elasticsearch/pulls/78724"
+          },
+          "html": {
+            "href": "https://github.com/elastic/elasticsearch/pull/78724"
+          },
+          "issue": {
+            "href": "https://api.github.com/repos/elastic/elasticsearch/issues/78724"
+          },
+          "comments": {
+            "href": "https://api.github.com/repos/elastic/elasticsearch/issues/78724/comments"
+          },
+          "review_comments": {
+            "href": "https://api.github.com/repos/elastic/elasticsearch/pulls/78724/comments"
+          },
+          "review_comment": {
+            "href": "https://api.github.com/repos/elastic/elasticsearch/pulls/comments{/number}"
+          },
+          "commits": {
+            "href": "https://api.github.com/repos/elastic/elasticsearch/pulls/78724/commits"
+          },
+          "statuses": {
+            "href": "https://api.github.com/repos/elastic/elasticsearch/statuses/f1fd5119702f71238c2dfc422296b8ef1f7f6740"
+          }
+        },
+        "author_association": "MEMBER",
+        "auto_merge": null,
+        "active_lock_reason": null,
+        "merged": false,
+        "mergeable": null,
+        "rebaseable": null,
+        "mergeable_state": "draft",
+        "merged_by": null,
+        "comments": 0,
+        "review_comments": 0,
+        "maintainer_can_modify": true,
+        "commits": 1,
+        "additions": 2,
+        "deletions": 2,
+        "changed_files": 1
+      }
+    },
+    "public": true,
+    "created_at": "2021-10-05T22:34:15Z",
+    "org": {
+      "id": 6764390,
+      "login": "elastic",
+      "gravatar_id": "",
+      "url": "https://api.github.com/orgs/elastic",
+      "avatar_url": "https://avatars.githubusercontent.com/u/6764390?"
+    }
+  }
+  ```
+</details>
+
+### Track Parameters
+
+Track parameters are specified using `--track-params`; e.g., `--track-params="bulk_size:1000,ingest_percentage:75"`
+
+| Parameter | Default | Description |
+| --- | --- | --- |
+| `bulk_size` | `5000` | Number of batched documents per bulk request |
+| `bulk_indexing_clients` | `8` | Number of clients issuing bulk indexing requests |
+| `codec` | `default` | The index compression codec to use. Use `best_compression` for higher compression at the cost of CPU. |
+| `ingest_mode` | `index` | Set to `data_stream` to index to a data stream, otherwise, use a standard index. |
```

Comment:
Providing a choice to use index or data streams leaves it up to the user if a search or metrics-oriented benchmark is desired. The idea is to offer the most flexibility. It could certainly default to `data_stream`.